alembic/versions/007_skeldir_foundation/202602141200_b07_p6_complexity_router_fields.py

```
@@ -0,0 +1,80 @@
+"""B0.7-P6 complexity router ledger fields.
+
+Revision ID: 202602141200
+Revises: 202602121210
+Create Date: 2026-02-14 12:00:00
+"""
+
+from typing import Sequence, Union
```

Ruff (0.15.0)

[warning] Import from `collections.abc` instead: `Sequence`

Import from `collections.abc`

(UP035)